pkg/controller/composite/controller.go

```diff
@@ -169,21 +171,26 @@ func newParentController(
 			return nil, err
 		}
 	}
+	parentIgnoreStatusChanges, err := newCompositeIgnoreStatusChangesMap(resources, cc)
```

Comment:
do we need this ? I believe it can be took from `cc` ?

Reply:
probably not needed for this PR, since in cc there is only one parent, but this could be useful when I create a subsequent PR for children resources, wdyt?

Reply:
it is not needed, you can check it directly like https://github.com/metacontroller/metacontroller/blob/master/pkg/controller/composite/controller.go#L620